### DISCUSSION TITLE
1974-06-21-story/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#19] [General]
# 1974-06-21-story/

Lena kan haar poppen wel vergeten.

https://fanzoflenazavaroni.github.io/1974-06-21-story/

**Comment by StephenMolloy1** [comment]
I've bitten the bullet and translated it using Google translate, there is the odd questionable result but it is generally good.

Lena can forget about her dolls.

Only ten years young, very cute to see and a voice that roars like a hurricane. That's Lena Zavaroni the new singing prodigy. The record business has already thrown itself at her with full force, because gold can be earned with such a natural talent.
Father and Mother Zavaroni really thought about it for a long time. And only when it was assured that their little girl would not be plunged headlong into the music business, did they give their consent. But from that day on, it was a normal childhood for Lena. She was soon the world's sweetest child prodigy.
A record was quickly made in the record studio, "Ma, He's making eyes at me" and 300,000 copies flew out of stores within a few days. The British record boys rubbed their hands in delight.

By the way, Daddy and Mommy Zavaroni weren't dissatisfied either, because through the booming voice of their ten-year-old offspring the money started pouring in. Lena flew in and out of the studio, made one television program after the other and was allowed to make sweet trips to European countries in between. After all, this great talent should not only be saved for England, no, the mainland also had to kneel before the endearing prodigy. And so it is that from now on you will regularly see and hear dear little Lena on the tube. And now you should start shouting impatiently for hall performances, because our labor inspectorate is not so generous with that for prodigies. Just think of Hein and Wilma.
In England this is seen somewhat more broadly. The little girl can also be seen on television all the time. But because you can't immediately tear such a cutie out of the familiar world of her dolls and skipping rope, the English labor inspectorate has determined that Lena is not allowed to perform more than forty times a year. Furthermore, the little darling is also not allowed to hang out in the studios after ten o'clock at night.
And that's where the aftercare stops, because of course you can't keep going. After all, Lena also has her sweet caring parents.
	"A special talent"
"The big problem for Mr. and Mrs. Zavaroni was how to publicize their little miracle without harming her upbringing in any way. Rightly so, for they were extremely concerned about the development of Lena's career, " wrote Phonogram in a biography of the Scottish prodigy. However, Lena's parents got in touch with Philip Solomon, a manager who is trained to keep very young talents on the right track, Solomon said. "At first I didn't feel like starting something with such a young girl, but when I heard Lena sing, I realized I was dealing with a very special talent. I will guide her further."

She is indeed special, because the simple Lena, who says very sweet things about the Netherlands during a conversation, that she likes being here and things like that, the same Lena becomes a creepy personality as soon as she is behind the microphone.
There is no longer a girl who would much rather play with her dolls and romp in the street with her girlfriends, but a real star! A creature that blindly imitates what the grown-ups have pumped into her. Professional movements, mischievous steps behind the microphone, a lot of working with the arms and the occasional almost comical, but certainly not intended, mischievous look into the camera.
	Expensive house in London
The future looks healthy for Lena - financially at least - yes. But the commercials and record guys won't spare her, because she just happens to be ten. Multitudes of records will have to be filled from them and she will be dragged from one unit corner of Europe to another, living within the shackles of success.
On the other hand, there is a rapidly growing bank account belonging to papa Zavaroni, a man who used to have to be frugal as a sales representative. Lena never had to nag for a new doll for a long time, such a little extra is also welcome because Lena's golden voice made it possible for father and mother to buy a very expensive house in London. London has become their new base: that's where the studios are and that's where the big world begins for Lena.
Meanwhile, Lena's spiritual formation is not neglected either. Papa has sent her to the Italia Conti School, a famous London institution for rich kids, where she is taught the necessary knowledge through private tuition. After all, the ten-year-old prodigy cannot remain stupid...

Photograph caption.
Playing with balloons, blowing fun bubbles or just doing what she likes is no longer an option for little Lena. The balloons and soap bubbles can only be used in front of the photographer's camera. For the rest, Lena can forget that, prodigy as she is.

Photographs by Peter Mazel.

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for taking the time to do this and for sharing it with me and other visitors to this website.

Don't know if you seen the latest from [Newsbank Post - Dated the 15th June 2023](https://infoweb.newsbank.com/apps/news/openurl?ctx_ver=z39.88-2004&rft_id=info%3Asid/infoweb.newsbank.com&svc_dat=UKNB&req_dat=55CA6C602C984FD8A3DCC6AF6BF4AE70&rft_val_format=info%3Aofi/fmt%3Akev%3Amtx%3Actx&rft_dat=document_id%3Anews%252F1922B1D9F10C1760) and here is the link to the original [Express Article - Dated 14 June 2023](https://www.express.co.uk/entertainment/theatre/1780852/Lena-Zavaroni-opportunity-knock-hughie-green) it captured.

**Reply by StephenMolloy1** [reply]
Thanks, I hadn't caught the Newsbank post.  The "Walking past a chippie"  quote is a new one. The trouble with liars is that they struggle to remember which version of their truth they told last week. I suspect that Hughie Green tipped off Philip Solomon about a brilliant little singer auditioning in Glasgow, he then sent Tommy Scott to go find out about her.

**Reply by FanzOfLenaZavaroni** [reply]
I'm not sure why so many versions of how Lena was discover has came about. Makes me wonder why there is a need to lie about such a thing. Is it just so important too each of the individuals to be know as the person that discover Lena or is there a another reason for the misleading stories. So quick to want to be known as the discoverer of Lena not so quick to admit their exploitation of her.

So now the story is that Tommy Scott was walking pass the Zavaroni family chip shop when from the flat above he heard someone singing and so went into the chippy to find out who it was and that lead to him bring Phil Solomon to Rothesay where he heard Lena sing in the Rothesay Pavilion.

Hold on what happen to the Athletic Bar where Tommy Scott heard Lena singing. Oh scrap the above he was walking pass an unnamed chip shop when he heard Lena singing in the Athletic Bar which was in Gallowgate and nowhere near a chip shop. Oh scrap that as well I was walking pass a chippy and carried on until I turn a corner into Gallowgate where I came upon the Athletic Bar where I heard a singer. Yes! Yes, this version will do (I did not known fiction was so hard).

I think the truth is that Green, Solomon and Scott already had a plan in place for Lena that for reasons (money being one of them) they did not what the public at large to know about and so did not want to connect Green with the other two.
